diff --git a/scripts/reports/utils.py b/scripts/reports/utils.py
index 5751139..6fbf353 100644
--- a/scripts/reports/utils.py
+++ b/scripts/reports/utils.py
@@ -183,7 +183,10 @@ def generate_guideline_compliance_summary(output_directory, results_summary):
                 print(
                     "**Result**: " + ("Not compliant" if total_guidelines_violated > 0 else "Compliant"))
                 standard_pretty_name = {
-                    "cert": "CERT C++ 2016", "autosar": "AUTOSAR C++  R22-11, R21-11, R20-11, R19-11 and R19-03"}
+                    "cert": "CERT C++ 2016",
+                    "autosar": "AUTOSAR C++  R22-11, R21-11, R20-11, R19-11 and R19-03",
+                    "misra": "MISRA C++:2023",
+                }
                 print("**Coding Standards applied**: " + ", ".join([standard_pretty_name[standard_short_name]
                       for standard_short_name in results_summary.guideline_violation_count.keys()]))

